includes/frontend/class-ur-frontend-form-handler.php

```diff
@@ -164,7 +164,25 @@ private static function validate_form_data( $form_field_data = array(), $form_da
 			array_push( self::$response_array, __( 'Required form field not found.', 'user-registration' ) );
 		}
 
+		// Check if a required field is missing.
+		$missing_item = array_diff( $form_key_list, $form_data_field );
+
+		if ( count( $missing_item ) > 0 ) {
+
+			foreach ( $missing_item as $key => $value ) {
+
+				if ( $value == $form_field_data[ $key ]->general_setting->field_name ) {
+
+					if ( 'yes' === $form_field_data[ $key ]->general_setting->required ) {
+						$response = $form_field_data[ $key ]->general_setting->label . ' is required field';
```

Comment:
@iamprazol We have options set required field error message in a general setting. No need to hardcode here.

Reply:
@lekhnath005 Since the messages are different i hardcoded it.